File: src/main/java/space/rogi27/homabric/utils/HomePermissionObject.java
package space.rogi27.homabric.utils;

import me.lortseam.completeconfig.api.ConfigEntry;
import org.spongepowered.configurate.objectmapping.ConfigSerializable;

@ConfigSerializable
public class HomePermissionObject {
    @ConfigEntry(comment = "Sets the maximum amount of homes per player.")
    public int maxHomes = 2;
}

File: src/main/kotlin/space/rogi27/homabric/objects/HomePermissionObject.kt
package space.rogi27.homabric.objects

import org.spongepowered.configurate.objectmapping.ConfigSerializable
import org.spongepowered.configurate.objectmapping.meta.Comment

@ConfigSerializable
class HomePermissionObject {
    @Comment("Sets the maximum amount of homes per player.")
    var maxHomes = 2
}